packages/shorebird_cli/lib/src/executables/gradlew.dart

```diff
@@ -113,3 +117,24 @@ class Gradlew {
     return productFlavors;
   }
 }
+
+extension on String {
+  /// Returns true when the string starts with at least two upper case letters
+  ///
+  /// Gradle flavors that are not capital case will not be transformed
+  /// to camel case and should be used as is. So this method helps to identify
+  /// those cases.
+  ///
+  /// Example:
+  /// ```dart
+  /// 'Test'.startsWithUpperCaseLetters; // false
```

Comment:
This one is still a bit surprising to me. Maybe we call this `areFirstTwoLetterUppercase`? I realize it's startsWithUpperCase**Letters** and not startsWithUpperCase**Letter**, but the subtlety is kind of misleading.

Reply:
Yeah, makes sense, changed to the name you suggested!